README.md

```diff
@@ -1,2 +1,40 @@
-# gaia-automation
-Charter for the Gaia Automation team
+Gaia Automation
+===============
+
+### Mission
+
+The Gaia Automation team takes responsibility for issues at the
+intersection of Firefox OS and automation. We aim to make it easy for
+developers of the FxOS system frontend and core FxOS apps to
+
+1. use external code,
+2. build gecko profiles for different scenarios (dev, test, prod),
+3. and test their code with unit and UI tests.
```

Comment:
@dylano I was thinking more in terms of the flow that developers use fetch stuff => build profile => test the thing. Clearer commitment sounds good.